graph/collection-properties.md

```diff
@@ -0,0 +1,402 @@
+OData services treat collections of complex types differently than they treat collections of entity types. 
+This is due to the nature of entity types being "individually addressable" (they have some key which uniquely identifies them within the collection) while complex types are not individually addressable. 
```

Comment:
Is there anywhere in this topic were we want to say something about collections of primitive types?